templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -821,7 +821,8 @@
     "        pixel_cluster_mask,\n",
```

Comment:
<p>Since you change the default <code>cluster_type</code> to  <code>'cell'</code>, you should explicitly pass <code>cluster_type='pixel'</code> here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
different function